js-plugins/mcp/src/server.ts

```
@@ -0,0 +1,59 @@
+        text: JSON.stringify(result),
+      }]
+    }
+  })
+
+
+  server.tool("get-farm-resource-info", "Get graph information of a module, including importers, dynamicImports, importedBindings and exports,modules.", {
+    filepath: z.string()
+      .describe('The absolute filepath of the module'),
+  }, async ({ filepath }) => {
+
+    let resource = compiler.resourcesMap[filepath as keyof typeof compiler.resourcesMap] as Resource
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'resource' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)